RFCs/RFC_2.md

```diff
@@ -0,0 +1,48 @@
+# RFC: Branch and PR naming and tagging convention
+
+**Primary author(s)**: Giovan Angela
+
+**Created**: 10-10-2022
+
+**Last updated**: 10-10-2022
+
+**Approver(s)**:
+
+| Name | yes/not yet | date |
+| ---- | ----------- | ---- |
+|      | [ ]         |      |
+
+## Overview
+<!-- If someone only reads this far, what do you want them to know? -->
+Use one convention when naming branches or PR's.
+
+## Goal
+<!-- What problems are you trying to solve? What problems are you not trying to solve? -->
+The goal is to have one naming convention for PR's and branches such that it looks nice in git(hub).
+
+## Motivation
+<!-- What is the current state of the world? Why is this change being proposed? -->
+There are several types of names beings used without a clear structure. Some even use the name to indicate if it is an enchancement, feature, bugfix, etc... This does not look nice or organised.
```

Comment:
What I'm missing here is: why is the branch name looking "nice" important?

Reply:
Poor choice of words since my inspiration was gone at the moment.